.github/workflows/daily-test.yml

```diff
@@ -62,3 +73,46 @@ jobs:
               "Link": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}",
               "Status": "${{ steps.tests.outcome }}"
             }
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.ROLE_TO_ASSUME }}
+          aws-region: us-east-1
+
+      - name: Setup Node.js - 18.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
+
+      - name: Install Dependencies
+        run: |
+          # Required to bydpass issue with canvas dependency: https://github.com/Automattic/node-canvas/issues/1862
+          sudo apt-get install -y libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential g++
+
+      - name: Send Metric to CloudWatch
+        run: |
+          node -e "
+          const { CloudWatchClient, PutMetricDataCommand } = require('@aws-sdk/client-cloudwatch');
+          const client = new CloudWatchClient({ region: 'us-east-1' });
```

Comment:
nit: put region to an env var since used in L81 as well?

Reply:
Unless there are other comments, I'll just leave it hardcoded. It's a small file and the region is unlikely to be changed.